### ISSUE TITLE
Switch base translation language based on menu selection

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-iOS#307]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-iOS/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-iOS' [Code of Conduct](https://github.com/scribe-org/Scribe-iOS/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue is to implement the change to the base keyboard translation language that the user translates from. The related Scribe menu issue is #255. The designs for #255 are found in the [designs in Figma](https://www.figma.com/file/c8945w2iyoPYVhsqW7vRn6/scribe_public_designs?type=design&node-id=405-393&t=nCPFM9zwswgPWWI6-0) :) Once the user selects a new language for the given keyboard, they should then see this chosen language as their base language with the translation keyboard being for the base language.

Note that this issue is blocked by the following:
- #255
- https://github.com/scribe-org/Scribe-Data/issues/70
- #458 

### Contribution

Happy to support and work on this! This issue is relatively low priority for now as we need lots of other issues to be finished beforehand.

**Comment by andrewtavis** [comment]
@Jag-Marcel, same here, can you write in so we can assign :)

**Comment by Jag-Marcel** [comment]
Already got started on this a little, what's the status on the Scribe-data side of things? Do we have a system to translate between languages other than English?

**Comment by andrewtavis** [comment]
Ya all of the individual files are ready to be used. The last step for this is https://github.com/scribe-org/Scribe-Data/issues/70 as it would be best to harness Google Collab and its GPUs for this. If there's no progress on that, I can also do the test, or also maybe this is something we could look into if you'd have interest :)

**Comment by andrewtavis** [comment]
Closed by #476! Glad to have this all set up, @Jag-Marcel! 😊
